src/main/java/iroha/validation/validators/impl/SampleValidator.java

```diff
@@ -1,5 +1,25 @@
 package iroha.validation.validators.impl;
 
-public class SampleRule {
+import iroha.protocol.TransactionOuterClass.Transaction;
+import iroha.validation.rules.Rule;
+import iroha.validation.rules.impl.SampleRule;
+import iroha.validation.validators.Validator;
+import java.util.Collection;
+import java.util.Collections;
 
+public class SampleValidator implements Validator {
+
+  private Collection<Rule> rules;
+
+  public SampleValidator() {
+    this.rules = Collections.singletonList(new SampleRule());
```

Comment:
we can move `Collections.singletonList(new SampleRule())` to 12th line. By doing that, we can remove this constructor.

Reply:
The idea was to have several opportunities to instantiate this, added different constructors